libraries/python/src/internal/portfolio.py

```
@@ -0,0 +1,97 @@
+
+    short_sum = (
+        data.lazyframe.filter(pl.col("side") == "SHORT")
+        .select(pl.sum("dollar_amount"))
+        .collect()
+        .item()
+    )
+
+    total_sum = long_sum + short_sum
+
+    if abs(long_sum - short_sum) / total_sum > maximum_imbalance_percentage:
+        message = f"Expected long and short dollar amount sums to be within {maximum_imbalance_percentage * 100}%, found long: {long_sum}, short: {short_sum}"  # noqa: E501
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)